runtime/src/legacymm/cpp/Memory.cpp

```diff
@@ -1394,7 +1394,7 @@ inline void decrementRC(ContainerHeader* container) {
         container->setBuffered();
         if (state->toFree != nullptr) {
           state->toFree->push_back(container);
-          MEMORY_LOG("toFree is now %d\n", state->toFree->size())
+          MEMORY_LOG("toFree is now %lu\n", state->toFree->size())
```

Comment:
Shouldn't everything `…->size()` should be `%zu`? Since `size()` returns `size_t` for std containers.

Reply:
It was compiler suggestion to use here `%lu`, changed to `%zu`